package.json

```diff
@@ -44,11 +45,6 @@
     "testMatch": [
       "<rootDir>/test/**/*.test.(ts|js)"
     ],
-    "globals": {
-      "ts-jest": {
-        "skipBabel": true
-      }
-    },
```

Comment:
This flag is now called `babelConfig` and is disabled by default https://kulshekhar.github.io/ts-jest/user/config/babelConfig

Reply:
This was fixed recently in #876